native/package.json

```diff
@@ -119,7 +119,6 @@
     "jest-react-native": "^18.0.0",
     "jest-transform-stub": "^2.0.0",
     "jetifier": "^1.6.5",
-    "js-yaml": "^3.14.0",
```

Comment:
I'm not sure but maybe flat can be removed here. Same for web.

Reply:
Required here because its used in `native/tools/build-config`. Same for `decamelize`. Already not in the web package.json.

In fact, flat it is used in the locales repo already and hoisted, so it could be omitted for now. But I think its much clearer if we just leave it here.